modules/node-observability-scripting-cr.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-observability-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-observability-scripting-cr_{context}"]
+= Creating the Node Observability custom resource for scripting
+
+You must create and run the `NodeObservability` custom resource (CR) before you run the scripting. When you run the `NodeObservability` CR, it creates the necessary machine config and machine config pool CRs to enable the scripting on the worker nodes matching the `nodeSelector`.
+
+.Prerequisites
+* You have installed the Node Observability Operator.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Log in to the {product-title} CLI by running the following command:
++
+[source,terminal]
+----
+$ oc login -u kubeadmin https://<HOSTNAME>:6443
+----
+
+. Switch back to the `node-observability-operator` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc project node-observability-operator
+----
+
+. Create a CR file named `nodeobservability.yaml` that contains the following text:
++
+[source,yaml]
+----
+    apiVersion: nodeobservability.olm.openshift.io/v1alpha2
+    kind: NodeObservability
+    metadata:
+      name: cluster <1>
+    spec:
+      nodeSelector:
+        kubernetes.io/hostname: <node_hostname> <2>
+      type: scripting
+----
+<1> You must specify the name as `cluster` because there should be only one `NodeObservability` CR per cluster.
+<2> Specify the nodes on which the Node Observability agent must be deployed.
+
```

Comment:
<3>  type must be set to scripting (this will deploy the agent in scripting mode)

Reply:
Ack.